nixos/modules/services/cluster/dynomite.nix

```diff
@@ -0,0 +1,62 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.dynomite;
+  local_redis = any (value: value == "127.0.0.1:${toString config.services.redis.port}:1") cfg.settings.dyn_o_mite.servers;
+  configFile = pkgs.runCommand "dynomite.yaml" {} ''echo '${builtins.toJSON cfg.settings}' | ${pkgs.yq}/bin/yq -y . > $out'';
```

Comment:
I don't think this should be needed, JSON is a subset of YAML, so any JSON file is valid YAML. If this is your workaround then you should make a comment in code explaining why this is necessary.

Reply:
JSON is a subset of YAML 1.2 not YAML 1.1 so no. This is a way to convert json to a valid yaml 1.1 file